torchdynamo/optimizations/training.py

```diff
@@ -60,8 +60,23 @@ def __init__(self, gm: torch.fx.GraphModule, example_inputs):
             if node.target == torch._C._set_grad_enabled:
                 has_set_grad_enabled = True
 
+        import functorch._src.config
+
+        if functorch._src.config.use_functionalize:
```

Comment:
If this is set, should we disable the dynamo functionalization in normalize_ir?

Reply:
Hmm, probably. I'll read over normalization and if I agree I can disable it.

Reply:
So, it seems, not all of normalization (only Functionalization), and dropping Functionalization in FX will drop un inplacing module, but I think that's OK.